src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,29 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import jakarta.persistence.EntityNotFoundException;
+import java.util.Optional;
+import mate.academy.hibernate.relations.dao.ActorDao;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDao actorDao;
+
+    public ActorServiceImpl(ActorDao actorDao) {
+        this.actorDao = actorDao;
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return actorDao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        Optional<Actor> actor = actorDao.get(id);
+        if (actor.isEmpty()) {
+            throw new EntityNotFoundException("Actor with id " + id + " not found");
+        }
+        return actor.get();
     }
```

Comment:
Remember to add `catch` blocks for operations of all types on the DAO layer. While the `add` and `get` methods are being called on the DAO, there are no catch blocks to handle possible `DataProcessingException` that might be thrown from the DAO layer. It's important to handle these exceptions and possibly rethrow them as a different type of exception that is meaningful at the service layer.